build.js

```diff
@@ -51,16 +51,18 @@ await build({
         );
 
         // This file pulls in a number of dependencies, but we don’t really need it anyway.
-        onResolve({ filter: /^\.+\/(util\/)?log$/ }, ({ importer, path }) => {
-          if (importer.includes(`${sep}tailwindcss${sep}`)) {
+        onResolve({ filter: /^\.+\/(util\/)?log$/, namespace: 'file' }, ({ path, ...options }) => {
```

Comment:
the namespace will prevent a recursion. as we set it to something random like `noRecurse`
https://esbuild.github.io/plugins/#on-resolve-options